tests/php/bootstrap.php

```diff
@@ -117,4 +121,4 @@ function in_running_uninstall_group() {
 
 // Using the Speed Trap Listener provided by WordPress Core testing suite to expose
 // slowest running tests. See the configuration in phpunit.xml.dist
-require $test_root . '/includes/listener-loader.php';
+require $test_root . '/includes/speed-trap-listener.php';
```

Comment:
I believe this needs to be reverted back. Different versions of PHPUnit requires different speed-trap-listeners. The loader helps ensure the right one is loaded.

https://github.com/WordPress/wordpress-develop/blob/master/tests/phpunit/includes/listener-loader.php

PHPUnit 7 requires PHP 7.1, so one is loaded if testing on newer PHP; the other for PHP 7 or below.

Reply:
Yep, my mistake, needed to `svn up` my source tree